lib/slack-hook-handler.js

```diff
@@ -34,7 +36,7 @@ SlackHookHandler.prototype.handle = function(params) {
         if (this.rooms.knowsSlackChannel(params["channel_id"])) {
             var roomID = this.rooms.matrixRoomID(params["channel_id"]);
```

Comment:
If you're referencing properties literally like this you can just do `params.channel_id` - both forms are valid but because you *have* to do computed property keys in `obj[keyVar]` format, there's a nice(ish) distinction between the two (well-defined structs use `foo.bar` whereas lookup tables use `foo[barVar]`